bin/serve

```diff
@@ -7,4 +7,10 @@ bin/checkinputs "$@"
 export OX_PROJECT=$1
 echo Serving: $OX_PROJECT
 
-webpack-dev-server --progress --config webpack.config.js
+dashboardSocket=$(grep 'webpackDashboardSocketPort' $OX_PROJECT/configs/base.coffee | sed 's|.*: \(.*\)|\1|')
```

Comment:
Is there a better way to handle this? grepping files seems very fragile . Could the port be defined in this file and then passed into `base.coffee` as an ENV variable?